src/ansi.rs

```diff
@@ -770,10 +778,104 @@ pub enum CursorShape {
     Hidden,
 }
 
-/// Terminal modes.
+/// Wrapper for the public mode.
```

Comment:
This should reference what type of mode this is. "the public mode" isn't really descriptive.